smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -51,14 +51,16 @@ def setUp(self):
             self.session.set(['interfaces','bonding','bond1','member','interface','dum0'])
             self.session.set(['interfaces','bonding','bond1','member','interface','dum1'])
             self._members.append('bond1')
+            self._mirror_interfaces.append('bond1')
 
         self._options['br0'] = []
         for member in self._members:
             self._options['br0'].append(f'member interface {member}')
 
     def tearDown(self):
-        self.session.delete(['interfaces','bonding'])
-        self.session.delete(['interfaces','dummy'])
+        self.session.delete(['interfaces','bonding','bond1'])
```

Comment:
Why is this change required?

Reply:
In fact, it doesn't need to be modified here. However, in order to ensure that the redundant interface will not be deleted, it is improved by the way. Do you want to restore the original deletion instruction?

Reply:
In fact, there is no need to change it here. But in order to ensure that the redundant interface is not deleted, it is improved